lib/deimos/schema_class/base.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require 'json'
+
+module Deimos
+  module SchemaClass
+    # Base Class for Schema Classes generated from Avro.
+    class Base
+
+      # :nodoc:
+      def initialize(_)
```

Comment:
Does it work if you replace `_` with `*args`?

Reply:
gonna try!

Reply:
yeap sure does - gonna use `*_args`